### DISCUSSION TITLE
What compiler are these cpp files written for?

### DISCUSSION THREAD

**Discussion by yashiautomations** [Oxyenyos/legacy_cplusplus#5] [Q&A] [answered]
What compiler are these cpp files written for? I've trouble running these on gcc. The includes are all messed up

**Comment by theWhiteWulfy** [comment, marked as answer]
These C++ files were written for use in TurboC++ v3.0 and will not work without changes in modern compilers like GCC. Hence the term legacy in the name. It is specifically meant for students who are still forced to use Turbo C++ in some school curricula in India, and I have this repo hosted so my former teacher can hand out the link for students to download, and for those who aren't proficient, probably use these in practical exams.

**Reply by yashiautomations** [reply]
Thank You

**Reply by Welding-Torch** [reply]
Quite nicely maintained.
